sql-database/failover-groups/add-single-db-to-failover-group-az-cli.sh

```diff
@@ -0,0 +1,112 @@
+﻿#!/bin/bash
+# Set variables
+export subscriptionID=<SubscriptionID>
+export resourceGroupName=myResourceGroup-$RANDOM
```

Comment:
__Recommended__: Resource group names don't need to be unique, so you can avoid the use of `$RANDOM` here.

Reply:
In addition to the BOM, the use of the `-` character and `$RANDOM` here may have some additional UTF-8 cruft added by Windows. Running `dos2unix` is likely required - not sure if Notepad++ fixes these issues.